docker/cla_frontend.ini

```diff
@@ -12,3 +12,5 @@ buffer-size=32768
 post-buffering-bufsize=32768
 http-workers=5
 alarm=removefile cmd:rm /tmp/listen_queue_healthy
+alarm-backlog=removefile cmd:rm /tmp/listen_queue_healthy
+harakiri=20
```

Comment:
did we not want harakiri only if the alarm was triggered?

Reply:
@jogilder is it possible to tie `harakiri` to alarm?
My understand is that `harakiri` accepts a value in seconds and any requests that takes longer that value will be dropped and the worker recycled https://uwsgi-docs.readthedocs.io/en/latest/Glossary.html